### PR TITLE
BlockStates and BlockModels

### PR DESCRIPTION
**Pull request by Karang**
This PR is not complete, but made to kickstart the discussion about how to support the block models.

The block models are needed by prismarine-viewer to represent correctly the blocks and to assign the correct textures to them.

I've extracted the block states and the block models, by just joining all the individual json files together, into a blocks_states.json and a block_models.json.

As you can see, the resulting json are fairly large and hard to parse. The format is explained here https://minecraft.gamepedia.com/Model

What kind of preprocessing (if any) should we apply to the data in order to make them similar across all versions and more easy to use. (Hopefully it seems like there are no major change in format from 1.8.8 to 1.13, which indicates that all versions should probably be similar)

### CODE REVIEW

**Comment by Karang** [comment]
I've added the remaining files for other versions as well. If we don't want to do the preprocessing here, we could merge as it is.

**Comment by rom1504** [comment]
I think it's not that big.
In this repo I think we should be doing minimal preprocessing, the idea is mostly to copy/paste the files and maybe provide a few additional json to help find the files
It makes sense to build some preprocessing either in node-minecraft-assets, either in another module

**Comment by rom1504** [comment]
@Karang could you do the corresponding node-minecraft-assets PR to expose this content to node ?
